components/Item/Card.vue

```diff
@@ -261,8 +261,13 @@ export default Vue.extend({
       }
     },
     getImageType(): ImageType {
-      if (this.shape === 'thumb-card' && this.item.Type === 'Movie') {
-        return ImageType.Backdrop;
+      if (
+        this.shape === 'thumb-card' &&
+        !['CollectionFolder', 'Folder', 'UserView'].includes(
+          this.item?.Type || ''
+        )
+      ) {
+        return ImageType.Thumb;
```

Comment:
Why is this changed?

Reply:
Because that's what jf-web does and it's what makes the most sense. We need #520 to be merged in order to implement proper fallback for these, but this is how it should've worked from the start.

Reply:
This breaks all card images in next-up and resume. Surely we shouldn't merge it unless the fix/refactor is implemented as well

Reply:
Temporarily fixed in #526